### PR TITLE
Added commands gra-case-action , gra-case-action-anomaly & gra-investigate-anomaly-summary

### PR DESCRIPTION
**Pull request by gra-xsoar**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml [hunk 1]
```diff
@@ -681,6 +687,42 @@ script:
...
+    - contextPath: Gra.Highrisk.Users.high_risk
+      description: High Risk
+      type: string
+    - contextPath: Gra.Highrisk.Users.is_orphan
+      description: Is Orphan
+      type: string
+    - contextPath: Gra.Highrisk.Users.domain
+      description: Domain
+      type: string
+    - contextPath: Gra.Highrisk.Users.risk_score
```
Comment (Bargenish) [inline Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml:723]: Is there a difference between this field and the one in [#L660](https://github.com/demisto/content/blob/20e04ef44ae91a10fe80b86ca84a43f14a4ec1d5/Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml#L660)?

**Review by Bargenish — CHANGES_REQUESTED** [review]
@upbalshetwar, @gra-xsoar Thank you for your contribution!

- Please regenerate the README.md file after the changes.
- Raise the pack version and add release notes.  Read [this](https://xsoar.pan.dev/docs/documentation/release-notes) for more inforamtion.

See my comments below.

File: Packs/Gurucul/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
 #### Integrations
 ##### Gurucul-GRA
-- Removed `(Community Contribution)` from the integration name.
+- Removed `(Community Contribution)` from the integration name.
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/ReleaseNotes/1_0_1.md:3]: By mistake, we removed this file so after that added
and if you observe in the file no changes are available

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py [hunk 3]
```diff
@@ -96,7 +111,12 @@ def fetch_record_command(client: Client, url_suffix, prefix, key, params):
 
 
 def fetch_records(client: Client, url_suffix, prefix, key, params):
-    results = fetch_record_command(client, url_suffix, prefix, key, params)
+    results = fetch_record_command(client, url_suffix, prefix, key, params, None)
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py]: Removed none extra parameter as suggested

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml [hunk 4]
```diff
@@ -620,12 +623,15 @@ script:
     - contextPath: Gra.User.title
       description: Title
       type: string
-    - contextPath: Gra.User.joingDate
+    - contextPath: Gra.User.joiningDate
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml:626]: The prior column is provided wrongly so provided right column name

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml [hunk 5]
```diff
@@ -749,11 +791,156 @@ script:
       description: Per page record count
       defaultValue: "25"
     outputs:
-    - contextPath: Gra.User.Anomalies.anomaly_name
+    - contextPath: Gra.Anomalies.anomaly_name
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml]: Changes as per prior context path

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py [hunk 6]
```diff
@@ -304,6 +324,39 @@ def main() -> None:
...
+                    demisto.args().get('subOption') + '","caseComment":"' + demisto.args().get('caseComment') + '"}'
+            fetch_post_records(client, cases_url, 'Gra.Case.Action', 'caseId', params, post_url)
+
+        elif demisto.command() == 'gra-case-action-anomaly':
+            cases_url = '/cases/' + demisto.args().get('action')
+            if demisto.args().get('action') == 'riskAcceptCaseAnomaly':
+                post_url = '{"caseId":' + demisto.args().get('caseId') + ',"anomalyNames":' + \
+                           demisto.args().get('anomalyNames') + ',"subOption":"' + demisto.args().get('subOption') +\
+                           '","caseComment":"' + demisto.args().get('caseComment') + '","riskAcceptDate":"' + \
+                           demisto.args().get('riskAcceptDate') + '"}'
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py]: Did changes and stored in the argument variable

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py [hunk 7]
```diff
@@ -20,16 +20,31 @@
...
+            if isinstance(r, dict):
+                try:
+                    json_array.append(r)
+                except ValueError:
+                    demisto.error("data was not valid JSON")
+                if len(json_array) > 0:
+                    incidents.extend(json_array)
+            else:
+                if len(r) > 0:
+                    incidents.extend(r)
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py]: did changes as requested

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml (hunk 1 above)
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml]: Removed line

**Comment by Bargenish** [comment]
closing, changes were made in https://github.com/demisto/content/pull/12946

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py [hunk 8]
```diff
@@ -304,6 +324,39 @@ def main() -> None:
             anomaly_url = '/users/' + demisto.args().get('employee_id') + '/anomalies/'
             fetch_records(client, anomaly_url, 'Gra.User.Anomalies', 'anomaly_name', params)
 
+        elif demisto.command() == 'gra-case-action':
+            cases_url = '/cases/' + demisto.args().get('action')
+            if demisto.args().get('action') == 'riskManageCase':
+                post_url = '{"caseId":' + demisto.args().get('caseId') + ',"subOption":"' + \
+                           demisto.args().get('subOption') + '","caseComment":"' + demisto.args().get('caseComment') +\
+                           '","riskAcceptDate":"' + demisto.args().get('riskAcceptDate') + '"}'
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py]: Code fixed.
Please check in the new merge.

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml [hunk 9]
```diff
@@ -749,11 +791,156 @@ script:
...
+    arguments:
+    - name: modelName
+      required: true
+      description: Model Name
+    - name: fromDate
+      description: From Date ( yyyy-MM-dd )
+    - name: toDate
+      description: To Date ( yyyy-MM-dd )
+    outputs:
+    - contextPath: Gra.Investigate.Anomaly.Summary.Analytical Features
```
[the remark this answers is not in the document]
Comment (gra-xsoar) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml]: Code fixed.
Please check in the new merge.

**Comment by Bargenish** [comment]
https://github.com/demisto/content/pull/13138

**Pull request closed without merging**